### PR TITLE
update single drive setup error message

### PR DESCRIPTION
**Pull request by harshavardhana**
## Description
update single drive setup error message

## Motivation and Context
Remove older references to FS mode

## How to test this PR?
```
~ minio server http://localhost/tmp/fs
```

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Unit tests added/updated
- [ ] Internal documentation updated
- [ ] Create a documentation update request [here](https://github.com/minio/docs/issues/new?label=doc-change,title=Doc+Updated+Needed+For+PR+github.com%2fminio%2fminio%2fpull%2fNNNNN)

### CODE REVIEW

File: internal/config/errors.go [hunk 1]
```diff
@@ -192,11 +192,11 @@ Refer to the link https://github.com/minio/minio/tree/master/docs/erasure/storag
...
-		"Please check the FS endpoint",
-		`FS mode requires only one writable disk path
-Example 1:
+	ErrInvalidEndpoint = newErrFn(
+		"Invalid endpoint for single drive mode",
+		"Please check the endpoint",
+		`Single drive mode requires absolute path without hostnames
+Examples:
    $ minio server /data/minio/`,
 	)
```
Comment (ravindk89) [inline internal/config/errors.go]:
```suggestion
Examples:
   $ minio server /data/minio/ #Single Node Single Drive
   $ minio server /data-{1...4}/minio # Single Node Multi Drive`,
	)
```

File: internal/config/errors.go [hunk 2]
```diff
@@ -192,11 +192,11 @@ Refer to the link https://github.com/minio/minio/tree/master/docs/erasure/storag
...
 
-	ErrInvalidFSEndpoint = newErrFn(
-		"Invalid endpoint for standalone FS mode",
-		"Please check the FS endpoint",
-		`FS mode requires only one writable disk path
-Example 1:
+	ErrInvalidEndpoint = newErrFn(
+		"Invalid endpoint for single drive mode",
+		"Please check the endpoint",
+		`Single drive mode requires absolute path without hostnames
```
Comment (ravindk89) [inline internal/config/errors.go]:
```suggestion
		`Single-Node modes requires absolute path without hostnames:
```

File: cmd/endpoint.go [hunk 3]
```diff
@@ -592,14 +592,14 @@ func CreateEndpoints(serverAddr string, foundLocal bool, args ...[]string) (Endp
 			return endpoints, setupType, err
 		}
 		if endpoint.Type() != PathEndpointType {
-			return endpoints, setupType, config.ErrInvalidFSEndpoint(nil).Msg("use path style endpoint for FS setup")
+			return endpoints, setupType, config.ErrInvalidEndpoint(nil).Msg("use path style endpoint for single drive setup")
```
Comment (ravindk89) [inline cmd/endpoint.go]:
```suggestion
			return endpoints, setupType, config.ErrInvalidEndpoint(nil).Msg("use path style endpoint for single node setup")
```

**Review by ravindk89 — APPROVED** [review]
A few suggestions, take or leave any that do/don't make sense.

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-erasure.sh | :heavy_check_mark: |
| mint-dist-erasure.sh | :heavy_check_mark: |
| mint-compress-encrypt-dist-erasure.sh | :heavy_check_mark: |
| mint-pools.sh | :heavy_check_mark: |
Deleting image on docker hub
Deleting image locally

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-erasure.sh | :heavy_check_mark: |
| mint-dist-erasure.sh | :heavy_check_mark: |
| mint-compress-encrypt-dist-erasure.sh | :heavy_check_mark: |
| mint-pools.sh | :heavy_check_mark: |
Deleting image on docker hub
Deleting image locally

**Pull request merged**